_docs/development/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Overview
+order: 001
```

Comment:
That makes sense. Is it possible to do this based on the directory structure underneath docs/install/gcp/index.md and docs/install/local/index.md? If not, this is fine for now.